scm/driver/gitlab/content.go

```diff
@@ -20,23 +19,12 @@ type contentService struct {
 }
 
 func (s *contentService) Find(ctx context.Context, repo, path, ref string) (*scm.Content, *scm.Response, error) {
-	path = url.QueryEscape(path)
-	path = strings.Replace(path, ".", "%2E", -1)
-	endpoint := fmt.Sprintf("api/v4/projects/%s/repository/files/%s?ref=%s", encode(repo), path, ref)
-	out := new(content)
+	endpoint := fmt.Sprintf("api/v4/projects/%s/repository/files/%s/raw?ref=%s", encode(repo), url.QueryEscape(path), ref)
+	out := new(bytes.Buffer)
 	res, err := s.client.do(ctx, "GET", endpoint, nil, out)
-	raw, berr := base64.StdEncoding.DecodeString(out.Content)
```

Comment:
That is something you should keep, that lead to issues in the past

Reply:
agreed, we should keep this workaround in place. I honestly cannot remember why we needed it in the first place, but the last thing I want to do is change something and cause an outage for hundreds or even thousands of Drone installations.

Reply:
edit: I see that this switches the API which does not encode the file, eliminating the base64 weirdness all together. Are there any concerns with file size? The base64 encoded API presumably limits file size to 1MB similar to GitHub which inherently protects Drone (primary consumer of this library) from downloading large files into memory. Maybe we should add an `io.LimitReader` here to mirror the 1MB limit in GitHub?